lib/api/apiUtils/bucket/bucketDeletion.js

```diff
@@ -129,8 +129,11 @@ export function deleteBucket(bucketMD, bucketName, canonicalID, log, cb) {
         // delete a bucket even if there are ongoing multipart uploads.
         function deleteMPUbucketStep(next) {
             const MPUBucketName = `${constants.mpuBucketPrefix}${bucketName}`;
+            // check to see if there are any mpu overview objects (so ignore
+            // any orphaned part objects)
             return metadata.listObject(MPUBucketName,
-                { maxKeys: 1 }, log, (err, objectsListRes) => {
+                { maxKeys: 1, prefix: 'overview' }, log,
```

Comment:
because '/' is not a good splitter for s3 since any character is permitted in an object key.